tests/lava/proc/monitor/test_monitors.py

```diff
@@ -60,7 +60,7 @@ def test_proc_compiled_without_error(self):
         monitor = Monitor()
         c = Compiler()
         # Compiling should run without error
-        c.compile(monitor, Loihi1SimCfg())
+        c.compile(monitor, Loihi1SimCfg(select_tag='floating_pt'))
```

Comment:
I still think that this should not be required. It makes no sense to tag the `Monitor` Process Model as `floating_pt`.
It's fine if this is a workaround for the release but if we are indeed planning on redesigning the tag system, please add an issue on that.

Reply:
Will leave it for now. Let's change the Monitor later.